### PR TITLE
Rename `pow_receipt_` to `deposit_`

### PR DESCRIPTION
**Pull request by hwwhww**
1. Per #358 item (3) - `pow_receipt_` to `deposit_`
    - `POW_RECEIPT_ROOT_VOTING_PERIOD` => `DEPOSIT_ROOT_VOTING_PERIOD`
    - `CandidatePoWReceiptRootRecord` => `DepositRootVote`
    - `candidate_pow_receipt_root` => `deposit_root_vote`
    - `processed_pow_receipt_root` => `processed_deposit_root`
2. Refactor `### Deposit roots` a little.
3. Also update #358 item (7). Misc shortenings in state - `candidate_pow_receipt_roots` => `deposit_root_votes`

### CODE REVIEW

File: specs/core/0_beacon-chain.md [hunk 1]
```diff
@@ -388,7 +389,7 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
     'randao_commitment': 'hash32',
     # Initial custody commitment
     'custody_commitment': 'hash32',
-    # a BLS signature of this ``DepositInput``
+    # A BLS signature of this `DepositInput`
```
Comment (djrtwo) [inline specs/core/0_beacon-chain.md]: Don't we generally do double back ticks in comments?

File: specs/core/0_beacon-chain.md [hunk 2]
```diff
@@ -553,12 +554,12 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
...
 
-#### `CandidatePoWReceiptRootRecord`
+#### `DepositRootVote`
 
 ```python
 {
-    # Candidate PoW receipt root
-    'candidate_pow_receipt_root': 'hash32',
+    # Deposit root
+    'deposit_root_vote': 'hash32',
```
Comment (djrtwo) [inline specs/core/0_beacon-chain.md]: I would just name this `deposit_root`. It is the root that defines this `DepositRootVote`

File: specs/core/0_beacon-chain.md [hunk 3]
```diff
@@ -605,28 +606,32 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
...
 * `withdrawal_credentials[1:] == hash(withdrawal_pubkey)[1:]` where `withdrawal_pubkey` is a BLS pubkey
 
-We recommend the private key corresponding to `withdrawal_pubkey` be stored in cold storage until a withdrawal is required.
+The private key corresponding to `withdrawal_pubkey` will be required to initiate a withdrawal. It can be stored separately until a withdrawal is required, e.g. in cold storage.
 
-### `Eth1Deposit` logs
+### `Deposit` logs
 
-Every deposit, of size between `MIN_DEPOSIT` and `MAX_DEPOSIT`, emits an `Eth1Deposit` log for consumption by the beacon chain. The deposit contract does little validation, pushing most of the validator onboarding logic to the beacon chain. In particular, the proof of possession (a BLS12-381 signature) is not verified by the deposit contract.
+Every Ethereum 1.0 deposit, of size between `MIN_DEPOSIT` and `MAX_DEPOSIT`, emits an `Deposit` log for consumption by the beacon chain. The deposit contract does little validation, pushing most of the validator onboarding logic to the beacon chain. In particular, the proof of possession (a BLS12-381 signature) is not verified by the deposit contract.
```
Comment (djrtwo) [inline specs/core/0_beacon-chain.md]: "an" -> "a"
```suggestion
Every Ethereum 1.0 deposit, of size between `MIN_DEPOSIT` and `MAX_DEPOSIT`, emits a `Deposit` log for consumption by the beacon chain. The deposit contract does little validation, pushing most of the validator onboarding logic to the beacon chain. In particular, the proof of possession (a BLS12-381 signature) is not verified by the deposit contract.
```

File: specs/core/0_beacon-chain.md [hunk 4]
```diff
@@ -696,7 +701,7 @@ For a beacon chain block, `block`, to be processed by a node, the following cond
 
 * The parent block with root `block.parent_root` has been processed and accepted.
 * The node has processed its `state` up to slot, `block.slot - 1`.
-* The Ethereum 1.0 block pointed to by the `state.processed_pow_receipt_root` has been processed and accepted.
+* A Ethereum 1.0 block pointed to by the `state.latest_deposit_root` has been processed and accepted.
```
Comment (djrtwo) [inline specs/core/0_beacon-chain.md]: "a" -> "an"

```suggestion
* An Ethereum 1.0 block pointed to by the `state.latest_deposit_root` has been processed and accepted.
```

File: specs/core/0_beacon-chain.md (hunk 1 above)
Comment (hwwhww) [reply specs/core/0_beacon-chain.md]: The double backticks are only for docstring.

File: specs/core/0_beacon-chain.md (hunk 2 above)
Comment (hwwhww) [reply specs/core/0_beacon-chain.md]: Fixed.

**Pull request merged**